File: C#1/Arrays/PrimeNumbers/PrimeNumbers.cs
using System;

namespace PrimeNumbers
{
    class PrimeNumbers
    {
        static void Main()
        {
            int n = int.Parse(Console.ReadLine());
            bool[] arr = new bool[n+1];
            arr[0] = arr[1] = true;
            for (int i = 2; i <= Math.Sqrt(n); i++)
            {
                if (arr[i] == false)
                {
                    for (int j = i * i; j <= n; j += i)
                    {
                        arr[j] = true;
                    }
                }
            }
            
            for (int k = n; k >= 0; k--)
            {
                if (arr[k] == false)
                {
                    Console.WriteLine(k);
                    break;
                }
            }
        }
    }
}

File: JavaScript Fundamentals/Arrays/PrimeNumbers.js
function solve(args) {
    var n = +args[0];
    var arr = [n + 1];
    for(var i = 2; i <= n; i++){
        arr[i] = false;
    }
    arr[0] = arr[1] = true;
    var biggest = 0;
    for (var i = 2, len = Math.sqrt(n); i <= len; i++) {
        if (arr[i] == false) {
            for (var j = i * i; j <= n; j += i) {
                arr[j] = true;
            }
        }
    }
    for(var i = n; i >= 0; i--){
        if(!arr[i]){
            return i;
            break;
        }
    }
}